modules/ROOT/pages/quick-start.adoc

```diff
@@ -8,6 +8,17 @@ Anypoint Monitoring provides data on Mule apps that are running on either CloudH
 [NOTE]
 Functional Monitoring is covered in xref:api-functional-monitoring::index.adoc[API Functional Monitoring].
 
+== Runtime Version Requirements
```

Comment:
"Runtime Version Requirements"
Is this (Runtime Version) Requirements or Runtime (Version Requirements)?
Or do you mean something else, since there are no requirements listed in this section, given the way that the sentences are structured?

- "you can use" doesn't denote a requirement. If you want to recast this sentence as a requirement, change "you can use" to "use".
- You put both Mule 3 and Mule 4 together under "for complete functionality", which implies that "complete functionality" means the same thing for both Mule versions, which I don't think it does. I think you mean this: "If you are using Mule 3, use version 3.8.7 or newer. Versions 3.8.6 and older are not supported. To have access to all of the features in Mule 4, use versions 4.1.6 or newer. Older versions do not include enhanced metrics for API Analytics."
- "If you are using a Mule 4 version that is older than version 4.1.5, you cannot use enhanced metrics for API Analytics." doesn't denote a requirement. If you want to recast the sentence as a requirement (and don't want to use my revision in the previous bullet point), change it to this: "If you want to use enhanced metrics for API Analytics, you must use a version of Mule 4 that is more recent than 4.1.5." In fact, this requirement is a bit odd. Above, you say, "For complete functionality, use Mule 4.1.6 or newer." Then, you say that 4.1.5 doesn't support enhanced metrics. So, either 4.1.5 also doesn't support another feature in addition to enhanced metrics, or you should say above, "For complete functionality, use Mule 4.1.5 or newer."
- "Versions 3.8.6 or older are not supported" contradicts the part where you say "for complete functionality, use Mule 3 3.8.7 or newer." In fact, if the user wants to use Mule 3 at all, the user must use 3.8.7 or newer. I suggest using my revision in the second bullet point.